app/helpers/me/notices_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Me::NoticesHelper
+  def dissmiss_all_notices_button
+    link_to_remote_with_confirm(:dismiss_all_notices.t,
```

Comment:
yes. definitly. Being able to use some of the bootstrap js is part of hte motivation for moving to jquery.